tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -267,6 +267,27 @@ def _run(self):
                                     rebal_files=volume.rebal_files,
                                     rebal_data=volume.rebal_data
                                 ).save()
+                # If gstatus does not return any status details in absence
+                # of volumes, default set the status and utilization details
+                out_lines = stdout.split('\n')
+                connected = True
+                for index in range(4, len(out_lines) - 2):
+                    if out_lines[index].split('\t')[2].strip() != 'Connected':
```

Comment:
I have seen output of gstatus is like this only always if volumes are not present.